datamade/ccfp-asset-dashboard
# Project form

## Overview
I've wired up a basic form for a Project and a need review on the code implementation. I've created views and URL routes for this interaction. I've also written some tests. There is no styling. 

This is for issue #12 but doesn't close it.

### Demo
![add-project-demo](https://user-images.githubusercontent.com/38969506/102635850-7f42bc80-4119-11eb-9e9b-6dc3f7ed1879.gif)

### Notes
The form only has two fields, which is incomplete for the Project model. I wanted to go ahead and submit a PR so I can get a check on implementation before I proceed with the rest of the model's fields.

## Testing Instructions
- Visit the `/projects/` route via the review app deployment: https://ccfp-asset-d-project-fo-t1y7ht.herokuapp.com/projects/
- Y should see a Project list view.
- Click "add project" and submit the form
- You should be redirected to the new project's detail view (it's a very basic view)
- Go back to the `/projects/` list view (there is no back button and "home" goes to another view. I will address with issue #15.)
- Your new project should be in the list view
